tslint.json

```diff
@@ -38,6 +37,6 @@
     "semicolon": [true, "always", "ignore-bound-class-methods"],
     "switch-default": true,
     "triple-equals": [true, "allow-null-check"],
-    "variable-name": [true, "check-format", "allow-leading-underscore", "ban-keywords"]
+    "variable-name": [true, "check-format", "allow-leading-underscore", "allow-pascal-case", "ban-keywords"]
```

Comment:
Any chance we could allow this only for icons? It saved me a couple times on our Angular components when declaring constants for providers.

Reply:
Yup makes sense and I was prob being too lazy. I'll change to allow only for icons.